src/io/OhmmsData/XMLParsingString.h

```diff
@@ -50,41 +89,60 @@ class XMLNodeString : public std::string
   }
 };
 
-/** convert an xmlNode attribute into a std::string
+/** convert an xmlNode attribute into a name value pair.
+ * used to inherit from std::string but to no advantage
+ * and repeated confusion about whether comparisons to an attribute string were between
```

Comment:
This is part of the commit log or PR description but not class documentation. It is only understandable when doing diff.